circuits/kimchi/src/polynomials/chacha.rs

```diff
@@ -514,7 +514,7 @@ mod tests {
             h.insert(Column::Index(GateType::Generic));
             h
         };
-        let mut powers = 0..20;
+        let mut powers = 0..24;
```

Comment:
It would be cool if we could do something like `chacha::constraint_count()` or something like that.